test/integration/hotrestart_handoff_test.py

```diff
@@ -58,7 +61,11 @@ def random_loopback_host():
 # the socket path too long.
 SOCKET_PATH = f"@envoy_domain_socket_{os.getpid()}"
 SOCKET_MODE = 0
-ENVOY_BINARY = "./test/integration/hotrestart_main"
+ENVOY_BINARY = ""  # Populated from args
```

Comment:
are these unused now? if so lets remove its kinda confusing trying to understand what is used where with these here

Reply:
if these "CONSTANTS" are getting written somewhere and then implicitly used with updated values then even more this needs fixing